lyricsgenius/song.py

```diff
@@ -140,7 +140,7 @@ def save_lyrics(self, filename=None, extension='json', verbose=True,
 
         # Write the lyrics to either a .json or .txt file
         if write_file:
-            with open(filename, 'wb' if binary_encoding else 'w') as lyrics_file:
+            with open(filename, 'wb' if binary_encoding else 'w', encoding='utf-8') as lyrics_file:
                 if extension == 'json':
                     json.dump(lyrics_to_write, lyrics_file)
```

Comment:
Is there a reason you didn't add the `ensure_ascii=False` parameter?

Reply:
> Is there a reason you didn't add the `ensure_ascii=False` parameter?

I'm sorry, I forgot to add it!

Reply:
Sorry for the wait, I added the missing parameter.